content/source/docs/enterprise/private/logging/index.html.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: "enterprise2"
+page_title: "Terraform Enterprise Logging"
+sidebar_current: "docs-enterprise2-home"
+---
+
+# Terraform Enterprise Logs
+
+This document contains information about interacting with Private Terraform Enterprise logs.
+
+# Application-level Logs
+
+Private Terraform Enterprise's application-level services all log to CloudWatch logs, with one stream per service. The stream names take the format:
+
+```
+{hostname}-{servicename}
+```
+
+Where `hostname` is the fqdn you provided when setting up Private Terraform Enterprise, and `servicename` is the name of the service whose logs can be found in the stream. More information about each service can be found in [`tfe-architecture`](#private-terraform-enterprise-architecture).
+
+For example, if your Private Terraform Enterprise installation is available at `tfe.mycompany.io`, you'll find CloudWatch Log streams like the following:
+
+```
+tfe.mycompany.io-atlas-frontend
+tfe.mycompany.io-atlas-worker
+tfe.mycompany.io-binstore
+tfe.mycompany.io-logstream
+tfe.mycompany.io-packer-build-manager
+tfe.mycompany.io-packer-build-worker
+tfe.mycompany.io-slug-extract
+tfe.mycompany.io-slug-ingress
+tfe.mycompany.io-slug-merge
+tfe.mycompany.io-storagelocker
+tfe.mycompany.io-terraform-build-manager
+tfe.mycompany.io-terraform-build-worker
+tfe.mycompany.io-terraform-state-parser
+```
+
+CloudWatch logs can be searched, filtered, and read from either from the AWS Web Console or (recommended) the command line [`awslogs`](https://github.com/jorgebastida/awslogs) tool.
+
+---
+
+## Audit Logs
+
+As of Private Terraform Enterprise release v201802-1, audit logging is available in Private Terraform Enterprise. 
+
+### Log Contents
+
+The audit log will be updated when any resource managed by Terraform Enterprise is changed. Read requests will be logged for resources deemed sensitive. These include:
+
+  * Authentication Tokens
+  * Configuration Versions
+  * Policy Versions
+  * OAuth Tokens
+  * SSH Keys
+  * State Versions
+  * Users
+  * Variables
+
+
+When requests occur, these are the pieces of information that will be logged:
+
+  1. The actor
+		* Users (including IP address and token used)
```

Comment:
We actually aren't logging the token used yet; this proved to be a bit more difficult and not as useful, as tokens are ultimately only visible to the user who owns them. IP address will be in there, though!